src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -37,6 +41,28 @@ public static Index parseIndex(String oneBasedIndex) throws ParseException {
         return Index.fromOneBased(Integer.parseInt(trimmedIndex));
     }
 
+    /**
+     * Parses {@code oneBasedIndexes} into a list of {@code Index} and returns it. Leading and trailing whitespaces will
+     * be trimmed.
+     *
+     * @throws ParseException if the specified index is invalid (not non-zero unsigned integer).
+     */
+    public static ArrayList<Index> parseMultipleIndexSeperatedByComma(String oneBasedIndexes) throws ParseException {
```

Comment:
Small nit but you could name this `parseCommaSeperatedIndexes` for clarity?

Reply:
Change method name to `parseCommaSeperatedIndexes` for clarity!